html/_sources/files/setup_docker.rst.txt

```diff
@@ -0,0 +1,141 @@
+.. _setup_docker:
+
+#############################################
+Simulated robot on docker with X11 forwarding
+#############################################
+
+This setup tutorial will guide you through setting up your PC to run the simulated robot with the demos.
+This assumes that you have a linux machine. Tested on ubuntu 20, but should work on any linux.
+
+Installing Docker
+-----------------
+
+Quick version:
+
+.. code-block:: bash
+
+    sudo apt install curl
+    echo "Installing docker"
+    curl -fsSL https://get.docker.com | sudo sh
+
+    groupadd docker
+    usermod -aG docker $(whoami)
+
+Slow version is to follow the instructions `here <https://docs.docker.com/engine/install/ubuntu/>`__.
+
+Cloning Robotont's packages
+-----------------------------
+
+All Robotont's packages can be accessed from `Robotont's GitHub <https://github.com/robotont>`__.
+
+Packages necessary to run the Gazebo simulation with Robotont's demos are following:
+
+#. `robotont_description <https://github.com/robotont/robotont_description>`__
+
+#. `robotont_nuc_description <https://github.com/robotont/robotont_nuc_description>`__
+
+#. `robotont_gazebo <https://github.com/robotont/robotont_gazebo>`__
+
+#. `robotont_navigation <https://github.com/robotont/robotont_gazebo>`__
+
+#. `robotont_demos <https://github.com/robotont/robotont_demos>`__
+
+#. `robotont_msgs <https://github.com/robotont/robotont_msgs.git>`__
+
+.. code-block:: bash
+
+    mkdir -p robotont/src
+    cd robotont/src
+
+    git clone https://github.com/robotont/robotont_description.git
+    git clone https://github.com/robotont/robotont_demos.git
+    git clone https://github.com/robotont/robotont_gazebo.git
+    git clone https://github.com/robotont/robotont_msgs.git
+    git clone https://github.com/robotont/robotont_navigation.git
+    git clone https://github.com/robotont/robotont_nuc_description.git
+
+    cd ..
+    pwd
+
+
+Instructions when using an nvidia GPU
+-------------------------------------
+Install nvidia-docker 2 `here <https://github.com/nvidia/nvidia-docker/wiki/Frequently-Asked-Questions#setting-up>`__.
+
+Use docker-ros-box to setup ros
+
+.. code-block:: bash
+
+    git clone https://github.com/KingBoomie/docker-ros-box.git
+    cd docker-ros-box
+    ./init-ros-box.sh melodic <PATH_RETURNED_BY_PWD_ABOVE>
```

Comment:
I'd maybe change <PATH_RETURNED_BY_PWD_ABOVE> to <PATH_TO_ROBOTONT_WORKSPACE> or something similar, since I skipped the cloning Robotont's packages parts and therefore didn't see that you had to run the pwd command in there.